baus/models.py

```diff
@@ -373,33 +343,24 @@ def household_relocation(households, household_relocation_rates,
 # demolish in the csv file - this also allows building multiple buildings and
 # just adding capacity on an existing parcel, by adding one building at a time
 @orca.step()
-def scheduled_development_events(buildings, development_projects,
-                                 demolish_events, summary, year, parcels,
-                                 mapping, years_per_iter, parcels_geography,
-                                 building_sqft_per_job, vmt_fee_categories,
-                                 static_parcels, base_year, scenario):
+def scheduled_development_events(buildings, development_projects, demolish_events, summary, year, parcels, mapping, years_per_iter, 
+                                 parcels_geography, building_sqft_per_job, vmt_fee_categories, static_parcels, base_year):
     # first demolish
     # 6/3/20: current approach is to grab projects from the simulation year
     # and previous four years, however the base year is treated differently,
     # eg 2015 pulls 2015-2010
     # this should be improved in the future so that the base year
```

Comment:
There is not! I think only I was bothered by it at the time. I think it still is applicable? It could be added to our base year update tasks depending on what you think.

Reply:
Let's capture / add as a task then we can triage for relative importance.

Reply:
Add a task here!: https://app.asana.com/0/385259290425521/1203858428142086